refreshable/src/main/java/com/palantir/refreshable/DefaultRefreshable.java

```diff
@@ -46,7 +47,7 @@
  * purposes of side-effects), to ensure that chains of unused derived Refreshables can be garbage collected, but any
  * undisposed side-effect subscribers keep all their ancestors alive.
  */
-final class DefaultRefreshable<T> implements SettableRefreshable<T> {
+final class DefaultRefreshable<@DoNotLog T> implements SettableRefreshable<@DoNotLog T> {
```

Comment:
The added annotations prevent regressions

second annotation wasn't necessary here
```suggestion
final class DefaultRefreshable<@DoNotLog T> implements SettableRefreshable<T> {
```